src/main/kotlin/org/britmoji/yoodle/extensions/LinkReplacerExtension.kt

```diff
@@ -41,7 +41,11 @@ class LinkReplacerExtension(override val name: String = "Link Replacer") : Exten
 
                 // Create
                 event.message.removeEmbeds()
-                channel.sendWebhook(event.member?.tag, event.member?.let { it.memberAvatar?.url ?: it.avatar?.url }) {
+                channel.sendWebhook(
+                    // hacky username fix
+                    event.member?.tag?.let { if (it.endsWith("#0")) it.dropLast(2) else it },
```

Comment:
Seems like a bit of a round-about way to do this. Can't you just replace("#0", "")?

Reply:
no. some people might have discrim still that starts with #0 (eg Derek#0001)